docker-entrypoint.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/sh
 
+# caddy_template is by default set to the backup to avoid adding the snippet
+# multiple times to the same template, for example when a container restarts
+export CADDY_TEMPLATE="${CADDY_TEMPLATE:-./docker-gen/templates/Caddyfile.bkp}"
+export CADDY_SNIPPET="${CADDY_SNIPPET}"
+
 set -o errexit
 set -o nounset
 
 # Create initial configuration:
+if [[ -z "$CADDY_SNIPPET" ]]; then
```

Comment:
``[[ -z`` is a bashism, but we are using `sh` 
We can try to use http://manpages.ubuntu.com/manpages/bionic/man1/checkbashisms.1.html